.github/workflows/tests.yml

```diff
@@ -0,0 +1,80 @@
+name: Tests
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install micromamba
+      shell: bash -l {0}
+      run: |
+        if [ "$RUNNER_OS" == "Linux" ]; then
+          wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
+        else
+          wget -qO- https://micromamba.snakepit.net/api/micromamba/osx-64/latest | tar -xvj bin/micromamba
+        fi
+        mv bin/micromamba ./micromamba
+        ./micromamba shell init -s bash -p ~/micromamba
+        mkdir -p ~/micromamba/pkgs/
+
+    - name: Mamba create env
+      shell: bash -l {0}
+      run: |
+        export MAMBA_ROOT_PREFIX=~/micromamba
+        export MAMBA_EXE=$(pwd)/micromamba
+        . $MAMBA_ROOT_PREFIX/etc/profile.d/mamba.sh
+        ./micromamba create --yes --name FOLIUM python=${{ matrix.python-version }} pip --file requirements.txt --file requirements-dev.txt --channel conda-forge
+
+    - name: Install folium
+      shell: bash -l {0}
+      run: |
+        export MAMBA_ROOT_PREFIX=~/micromamba
+        export MAMBA_EXE=$(pwd)/micromamba
+        . $MAMBA_ROOT_PREFIX/etc/profile.d/mamba.sh
+        micromamba activate FOLIUM
+        pip install -e . --no-deps --force-reinstall
+
+    - name: Tests
+      shell: bash -l {0}
+      run: |
+        export MAMBA_ROOT_PREFIX=~/micromamba
+        export MAMBA_EXE=$(pwd)/micromamba
+        . $MAMBA_ROOT_PREFIX/etc/profile.d/mamba.sh
+        micromamba activate FOLIUM
+        pytest -vv --ignore=tests/selenium
+
+    - name: Selenium tests
+      shell: bash -l {0}
+      run: |
+        export MAMBA_ROOT_PREFIX=~/micromamba
+        export MAMBA_EXE=$(pwd)/micromamba
+        . $MAMBA_ROOT_PREFIX/etc/profile.d/mamba.sh
+        micromamba activate FOLIUM
+        pytest -vv --ignore=tests/selenium
+
+    - name: Tarball tests
+      shell: bash -l {0}
+      run: |
+        export MAMBA_ROOT_PREFIX=~/micromamba
+        export MAMBA_EXE=$(pwd)/micromamba
+        . $MAMBA_ROOT_PREFIX/etc/profile.d/mamba.sh
+        micromamba activate FOLIUM
```

Comment:
This repetition is b/c GH actions seems to start a new shell for every section.
A micromamba GH Actions would solve this. We should create one as soon as they issue a new release with some bugfixes and new features that will make this easier.